resources/lib/youtube_plugin/kodion/constants/__init__.py

```diff
@@ -63,6 +63,7 @@
 PLAYBACK_STOPPED = 'playback_stopped'
 REFRESH_CONTAINER = 'refresh_container'
 RELOAD_ACCESS_MANAGER = 'reload_access_manager'
+PLAYBACK_FAILED = 'playback_failed'
```

Comment:
Can you keep this group of events in alphabetical order? And the `__all__` tuple too.